apps/labs/posts/jupyter-community-calls.md

```diff
@@ -0,0 +1,120 @@
+---
+title: 'A year of Jupyter community calls'
+published: 2021-12-10
+author: Isabela Presedo-Floyd
```

Comment:
```suggestion
author: isabela-presedo-floyd
```
This blog post formats names differently from other converted blog posts. In a [previous blog post PR with Isabela](https://github.com/Quansight/Quansight-website/pull/365/files#diff-40664df3f220a56b3209d6ec2e3086e93ed169724e8ffcea957bb287e3dfa0e9R4), her name is formatted as isabela-presedo-floyd. I think the latter formatting is more consistent with the blog tagging system.

I'm not sure if I should change the name formatting, so I'm putting this as a Github Suggestion.

For future blog PRs, should I switch the name formatting myself, before making a PR?